src/main/java/com/marklogic/kafka/connect/sink/DefaultSinkRecordConverter.java

```diff
@@ -1,20 +1,45 @@
 package com.marklogic.kafka.connect.sink;
 
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Collections;
+import java.util.Date;
+import java.util.Map;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Struct;
+/*For Avro conversion */
+import org.apache.kafka.connect.json.JsonConverter;
+import org.apache.kafka.connect.sink.SinkRecord;
+import org.apache.kafka.connect.storage.Converter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import com.marklogic.client.document.DocumentWriteOperation;
 import com.marklogic.client.ext.document.DocumentWriteOperationBuilder;
 import com.marklogic.client.io.BytesHandle;
 import com.marklogic.client.io.DocumentMetadataHandle;
 import com.marklogic.client.io.Format;
 import com.marklogic.client.io.StringHandle;
 import com.marklogic.client.io.marker.AbstractWriteHandle;
+import org.apache.kafka.connect.json.JsonConverter;
 import org.apache.kafka.connect.sink.SinkRecord;
-import java.util.Map;
+import org.apache.kafka.connect.storage.Converter;
+import java.util.Collections;
+import org.apache.kafka.connect.json.JsonConverter;
 
 /**
  * Handles converting a SinkRecord into a DocumentWriteOperation via the properties in the given config map.
  */
 public class DefaultSinkRecordConverter implements SinkRecordConverter {
 
+	String converter = "STRING"; 
```

Comment:
This doesn't seem safe, as the value of this is being changed. Whereas all the other fields are initialized in the constructor and then remain static. Is MarkLogicSinkTask guaranteed to be single-threaded, such that its "put" method will never be invoked at the same time by two or more threads? Because if the latter happens, this will break. 

Actually, what's this field even used for? I found places where it's set, but I didn't find anywhere that its value is used.